### ISSUE TITLE
Error on refresh API

### ISSUE THREAD

**Issue by MarStefo** [open-wa/wa-automate-nodejs#2312]
Hi, when I run `http://localhost:8080/refresh` from the API, I get the following response:

```
{
  "success": false,
  "error": {
    "name": "Error",
    "message": "Evaluation failed: TypeError: Cannot read property 'sendMessage' of undefined\n    at __puppeteer_evaluation_script__:1:117221\n    at __puppeteer_evaluation_script__:1:450370"
  }
}
```

Also, I am unable to send messages (EDIT: it works), but I think it is due to the previous error.

<img width="1066" alt="Screenshot 2021-11-11 at 12 36 39" src="https://user-images.githubusercontent.com/6590445/141291617-4a3ddde3-2860-4c2f-9c79-49ab00170ce2.png">

**Comment by smashah** [comment]
@github-actions run

<details>
<summary>⚡ Release!  ⚡</summary>

```js
(async () => {
function exec(cmd) {
  console.log(execSync(cmd).toString());
}
//set the version type
process.env.VERS = "patch"

// Config
const gitUserEmail = "github-actions[bot]@users.noreply.github.com";
const gitUserName = "github-actions[bot]";

exec(`echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc`);
exec(`git config --global user.email "${gitUserEmail}"`);
exec(`git config --global user.name "${gitUserName}"`);
exec(`npm i -D`);
exec(`npm run release-ci $VERS`);

// types only package
exec('npm config set //registry.npmjs.org/:_authToken $NPM_TOKEN')
exec(`cd ./types-only && npm version $VERS && npm run build && npm publish && cd ..`);
exec(`git commit -a -m 'updated types-only package'`);
exec(`git push --force`);

//comment on the issue
var result = execSync(`npx auto-changelog -o ./tempchangelog.txt --commit-limit false --template ./compact-keepachangelog.hbs --stdout`).toString();

    await postComment(result);

//create changelog image
exec(`npm run release-image`);
exec(`git commit -a -m 'updated release-image'`);
exec(`git push --force`);
  })();
```
</details>

**Comment by smashah** [comment]
# Changelog

## 🚀 Release [4.23.17](https://github.com/open-wa/wa-automate-nodejs/compare/4.23.16...4.23.17) (2021-11-21)

- 🐛 attempt to fix #2312 [`#2312`](https://github.com/open-wa/wa-automate-nodejs/issues/2312)
- Release 4.23.17 [`f553cbe`](https://github.com/open-wa/wa-automate-nodejs/commit/f553cbeed1bea10980d931e0545021426d9ea3e6)
- 🥚 patches update [`4150c9b`](https://github.com/open-wa/wa-automate-nodejs/commit/4150c9b36401089bcff138921a28870f560e0b61)
- 🥚 patches update [`f683a9d`](https://github.com/open-wa/wa-automate-nodejs/commit/f683a9d2294f278236178d2ce32a1e83d43e4be8)
- updated types-only package [`351cea6`](https://github.com/open-wa/wa-automate-nodejs/commit/351cea67bb8f40a55c25530a9ecb79e68d627f73)

**Comment by smashah** [comment]
This is fixed now in 4.23.17

https://hub.docker.com/layers/openwa/wa-automate/latest/images/sha256-4c2f60809039b7168035e7782108e1c765e7235d517940157eac08f74cb2cc42?context=repo

https://hub.docker.com/layers/178441695/openwa/wa-automate/4.23.17/images/sha256-4c2f60809039b7168035e7782108e1c765e7235d517940157eac08f74cb2cc42?context=repo

**Comment by MarStefo** [comment]
Wow super! Thank you @smashah !
